supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -52,15 +52,14 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
     case other => log.error(s"Unexpected message to JesPollingManager: $other")
   }
 
-  private def handleQueryFailure(failure: JesApiQueryFailed) = failure match {
-    case JesApiQueryFailed(request, _) if request.failedAttempts < maxRetries =>
-      val nextRequest = request.withFailedAttempt
-      val delay = nextRequest.backoff.backoffMillis.millis
-      context.system.scheduler.scheduleOnce(delay, self, nextRequest)
-      ()
-    case JesApiQueryFailed(request, _) =>
-      request.requester ! failure
-      ()
+  private def handleQueryFailure(failure: JesApiQueryFailed) = if (failure.query.failedAttempts < maxRetries) {
+    val nextRequest = failure.query.withFailedAttempt
+    val delay = nextRequest.backoff.backoffMillis.millis
+    context.system.scheduler.scheduleOnce(delay, self, nextRequest)
+    ()
+  } else {
+    failure.query.requester ! failure
+    ()
```

Comment:
Super nit pick: I don't think you need this `()`, the `!` already returns `Unit`